arcgis-ios-sdk-samples/Utility Network/Configure subnetwork trace/ConfigureSubnetworkTrace.storyboard

```diff
@@ -0,0 +1,277 @@
+                            <constraint firstItem="cNs-C6-zew" firstAttribute="leading" secondItem="bNM-9x-Os8" secondAttribute="leading" id="xjW-Ns-n17"/>
+                        </constraints>
+                        <viewLayoutGuide key="safeArea" id="Qfl-lK-vPg"/>
+                    </view>
+                    <connections>
+                        <outlet property="barriersSwitch" destination="wNQ-FV-0T2" id="yim-Se-fqH"/>
+                        <outlet property="containersSwitch" destination="DQh-jP-zoT" id="a7w-nn-Tv9"/>
+                        <outlet property="tableView" destination="AON-wr-WJ5" id="7kT-uO-sA3"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="6h4-6y-3he" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+                <switch opaque="NO" tag="1" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" id="DQh-jP-zoT" userLabel="Containers Switch">
+                    <rect key="frame" x="0.0" y="0.0" width="51" height="31"/>
+                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                </switch>
+                <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" id="wNQ-FV-0T2" userLabel="Barriers Switch">
+                    <rect key="frame" x="0.0" y="0.0" width="51" height="31"/>
+                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMaxY="YES"/>
+                </switch>
+            </objects>
+            <point key="canvasLocation" x="3500.0000000000005" y="85.714285714285708"/>
+        </scene>
+        <!--Add condition-->
+        <scene sceneID="cZu-hF-J2e">
+            <objects>
+                <tableViewController id="YFg-XB-smY" customClass="ConfigureSubnetworkTraceOptionsViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="static" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" id="xZ5-pv-0lh">
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="842"/>
+                        <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <sections>
+                            <tableViewSection headerTitle="Condition" id="WQe-NV-8Gg">
```

Comment:
Since the title of the view is "Add Condition", I don't think this section needs a title.